velox/docs/functions/spark/datetime.rst

```diff
@@ -7,6 +7,11 @@ Convenience Extraction Functions
 
 These functions support TIMESTAMP and DATE input types.
 
+.. spark:function:: add_months(startDate, numMonths) -> integer
+
+    Returns the date that is ``numMonths`` after ``startDate``.
+    ``numMonths`` can be zero to does nothing or negative to subtract months.
```

Comment:
typos: When numMonths is zero returns startDate unmodified. When numMonths is negative subtracts that many months from the startDate.